examples/aws/storage-efs/README.md

```diff
@@ -0,0 +1,225 @@
+```
+export CLUSTER_NAME=<clustername>
+export CLUSTER_REGION=<clusterregion>
+export AWS_ACCOUNT_ID=$(aws sts get-caller-identity --query "Account" --output text)
+```
+
+## 2.0 Install the EFS CSI Driver
+We recommend installing the EFS CSI Driver v1.3.4 directly from the [the aws-efs-csi-driver github repo](https://github.com/kubernetes-sigs/aws-efs-csi-driver) as follows - 
+
+```
+kubectl apply -k "github.com/kubernetes-sigs/aws-efs-csi-driver/deploy/kubernetes/overlays/stable/?ref=tags/v1.3.4"
+```
+
+You should see the following resources get created - 
+```
+serviceaccount/efs-csi-controller-sa created
+serviceaccount/efs-csi-node-sa created
+clusterrole.rbac.authorization.k8s.io/efs-csi-external-provisioner-role created
+clusterrolebinding.rbac.authorization.k8s.io/efs-csi-provisioner-binding created
+deployment.apps/efs-csi-controller created
+daemonset.apps/efs-csi-node created
+csidriver.storage.k8s.io/efs.csi.aws.com configured
+```
+
+Additionally, you can confirm that EFS CSI Driver was installed into the default kube-system namespace for you. You can check using the following command - 
+```
+kubectl get csidriver
+
+NAME              ATTACHREQUIRED   PODINFOONMOUNT   MODES        AGE
+efs.csi.aws.com   false            false            Persistent   5d17h
+```
+
+## 3.0 Create the IAM Policy for the CSI Driver
+The driver requires IAM permission to talk to Amazon EFS to manage the volume on user's behalf. Here, we will be creating/annotating the Service Account `efs-csi-controller-sa` with an IAM Role which has the required permissions.
+
+1. Download the IAM policy document from GitHub as follows - 
+
+```
+curl -o iam-policy-example.json https://raw.githubusercontent.com/kubernetes-sigs/aws-efs-csi-driver/v1.3.4/docs/iam-policy-example.json
+```
+
+2. Create the policy - 
+```
+aws iam create-policy \
+    --policy-name AmazonEKS_EFS_CSI_Driver_Policy \
+    --policy-document file://iam-policy-example.json
+```
+
+3. Create an IAM role and attach the IAM policy to it. Annotate the Kubernetes service account with the IAM role ARN and the IAM role with the Kubernetes service account name. You can create the role using eksctl as follows - 
+
+```
+eksctl create iamserviceaccount \
+    --name efs-csi-controller-sa \
+    --namespace kube-system \
+    --cluster $CLUSTER_NAME \
+    --attach-policy-arn arn:aws:iam::$AWS_ACCOUNT_ID:policy/AmazonEKS_EFS_CSI_Driver_Policy \
+    --approve \
+    --override-existing-serviceaccounts \
+    --region $CLUSTER_REGION
+```
+
+4. You can verify by describing the specified service account to check if it has been correctly annotated - 
+```
+kubectl describe -n kube-system serviceaccount efs-csi-controller-sa -n kube-system
+```
+
+## 4.0 Create an Instance of the EFS Filesystem
+This section creates a new EFS volume using for your cluster. Please refer to the official [AWS Documents](https://docs.aws.amazon.com/eks/latest/userguide/efs-csi.html) for more details. 
```

Comment:
This section creates a new EFS volume using for your cluster.

`This section creates a new EFS volume, which will be used by your cluster.`

Reply:
Following 4 steps looks like copy paste from aws doc...is it better to redirect user and provide precise statement on which all instructions to follow than copy-paste here?

It will be hard to maintain when things changes on EFS side?

Reply:
Exactly what I was trying to say on slack :)
However, in my opinion it is nicer to have a complete flow in a single document. While I think it is ok to depend on "clonable" source from a separate repo as we did for the EFS Driver (or as Suraj is recommending for the sample PV, PVC files), but we should not ask the user to hop through multiple guides. Thus, in this case we should just include the commands here itself.

Open to a different decision - final opinions @goswamig @surajkota ?

Reply:
I disagree, we are not asking user to switch console to terminal and vice versa. 
Its a different set of resources and managed by different team. This README will be outdated if you copy paste from aws doc and EFS team makes any change. You will not even know when such change happens and customer will have broken experience. Its better to have multiple hops than broken experience.

Reply:
Well both approaches have the same problem, some links(related to training script or dataset) in our operator documentation which were pointing to sagemaker docs were broken and we never came to know about it. 

It is hard to find a balance and I faced similar dilema while working on cognito readme. I chose to have user read aws doc where whole document needs to be followed (e.g. certificate creation)  vs where deployment needs things to be created in a specific fashion(e.g. app client setting or custom domain creation in cognito userpool)

so maybe use/clone if its a tag/commit 
for here, since we are pinning to a specific version which we tested 1.3.4, I think its ok to have these steps here(we also have the link just incase). Also, PVC namespace also need to be in user profile in Kubeflow.

Reply:
```Well both approaches have the same problem, some links(related to training script or dataset) in our operator documentation which were pointing to sagemaker docs were broken and we never came to know about it.
```
At least you can ask them to fix it. I am sure service owner team has this responsibility to fix it. Duplicate instruction is never a good idea..
Take a look in the other projects under awslab most of them redirect to aws doc. I dont understand why will there be a dilema .This is not a bug bash doc, but customer/user facing doc.

Reply:
ok, removing both section 4 and 5 and pointing to the official AWS Docs.

Reply:
I removed this section and pointed to the official docs.